### DISCUSSION TITLE
What kind of cluster needed

### DISCUSSION THREAD

**Discussion by brybacki** [docent-net/golearnistio#4] [Q&A]
What kind of cluster is needed for the experiments? Is something like minikube or CRC enough?
What is your setup?

**Comment by docent-net** [comment]
Hello Bartosz, kopę lat :)

Not sure about CRC, but generally OpenShift and Minikube should be just fine: https://istio.io/latest/docs/setup/platform-setup/

Here some write-up about Istio on Minikube: https://medium.com/@jobinesh/setting-up-istio-on-minikube-for-running-bookinfo-demo-application-af25dab2a732

I'm using bare-metal k8s installations. The only thing I'd be a bit afraid would be installing and configuring Kiali (which we will use for observability) on Minikube (or CRC). You'd need to verify whether it works.

**Reply by brybacki** [reply]
Found some info, will try it ;)
Installing Istio and Kiali on Minikube (in a few minutes):  https://www.youtube.com/watch?v=0Ho1twJ9Udg&feature=emb_title

BTW: do you have a multinode bare-metal setup?

**Reply by docent-net** [reply]
Exactly 2x3 = 6

HP thin-clients t620, bought from aftermarket (https://www8.hp.com/us/en/thin-clients/rd-poc/t620.html)

It works like a charm; extended ram up to 8gb per node (max 16gb) and SSD to ~16gb per node (but I use mainly persistent storage via NFS from Qnap)

**Comment by Dr4il** [comment]
Guys, according to k8s on bare metals, do you have any experience with
rook.io and ceph configuration?

W dniu pt., 29.01.2021 o 16:04 Maciej Lasyk <notifications@github.com>
napisał(a):

> Hello Bartosz, kopę lat :)
>
> Not sure about CRC, but generally OpenShift and Minikube should be just
> fine: https://istio.io/latest/docs/setup/platform-setup/
>
> Here some write-up about Istio on Minikube:
> https://medium.com/@jobinesh/setting-up-istio-on-minikube-for-running-bookinfo-demo-application-af25dab2a732
>
> I'm using bare-metal k8s installations. The only thing I'd be a bit afraid
> would be installing and configuring Kiali (which we will use for
> observability) on Minikube (or CRC). You'd need to verify whether it works.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/docent-net/golearnistio/discussions/4#discussioncomment-320974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG52FCMHIQIGK2HFOJMGAZTS4LFBLANCNFSM4WY7T6ZA>
> .
>
-- 
Pozdrawiam
Szymon Wolszczak

**Reply by docent-net** [reply]
> Guys, according to k8s on bare metals, do you have any experience with rook.io and ceph configuration? W dniu pt., 29.01.2021 o 16:04 Maciej Lasyk <notifications@github.com> napisał(a):
> Hello Bartosz, kopę lat :) Not sure about CRC, but generally OpenShift and Minikube should be just fine: https://istio.io/latest/docs/setup/platform-setup/ Here some write-up about Istio on Minikube: ***@***.***/setting-up-istio-on-minikube-for-running-bookinfo-demo-application-af25dab2a732 I'm using bare-metal k8s installations. The only thing I'd be a bit afraid would be installing and configuring Kiali (which we will use for observability) on Minikube (or CRC). You'd need to verify whether it works. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <[#4 (comment)](https://github.com/docent-net/golearnistio/discussions/4#discussioncomment-320974)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AG52FCMHIQIGK2HFOJMGAZTS4LFBLANCNFSM4WY7T6ZA> .
> -- Pozdrawiam Szymon Wolszczak

I don't have production exp w/rook or ceph on bare/k8s. However, you won't need it for this workshop ;)

**Reply by Dr4il** [reply]
Yea, just asking for some tips about bugs/problems. 
Anyway maybe better use kind instead of minikube?

**Reply by docent-net** [reply]
Actually, I leave the decision of the k8s installation to participants. There're just to many possibilities these days
